jwplayer-appletv-web-app/js/playlist/PlaylistLoader.js

```diff
@@ -89,7 +89,8 @@ var PlaylistLoader = function() {
         mediaItem.mediaid = playlistItem.mediaid;
         mediaItem.description = playlistItem.description;
         mediaItem.title = playlistItem.title;
-        mediaItem.artworkImageURL = _checkScheme(playlistItem.image);
+        mediaItem.artworkImageURL = playlistItem.image ?
+          _checkScheme(playlistItem.image) : "";
```

Comment:
@pablos1 Should we handle other fields in this manner too? It'll make playlist loading more robust, but we may up with unexpected behavior when fields are missing anyway.